api/v01/entities/vrp_result.rb

```diff
@@ -45,16 +46,18 @@ class VrpResultSolutionRouteActivityDetails < Grape::Entity
     end
 
     class VRPResultDetailedCosts < Grape::Entity
-      expose :total, documentation: { type: String, desc: 'Cumulated cost' }
-      expose :fixed, documentation: { type: String, desc: 'Cost associated to the use of the vehicle' }
-      expose :time, documentation: { type: String, desc: 'Cost associated to the time dimension' }
-      expose :distance, documentation: { type: String, desc: 'Cost associated to the distance dimension' }
-      expose :value, documentation: { type: String, desc: 'Cost associated to the value dimension' }
-      expose :lateness, documentation: { type: String, desc: 'Cost associated to late arrival' }
-      expose :overload, documentation: { type: String, desc: 'Cost associated to quantities overload' }
+      expose :total, documentation: { type: Float, desc: 'Cumulated cost' }
+      expose :fixed, documentation: { type: Float, desc: 'Cost associated to the use of the vehicle' }
+      expose :time, documentation: { type: Float, desc: 'Cost associated to the time dimension' }
+      expose :distance, documentation: { type: Float, desc: 'Cost associated to the distance dimension' }
+      expose :value, documentation: { type: Float, desc: 'Cost associated to the value dimension' }
+      expose :lateness, documentation: { type: Float, desc: 'Cost associated to late arrival' }
+      expose :overload, documentation: { type: Float, desc: 'Cost associated to quantities overload' }
```

Comment:
In fact https://github.com/Mapotempo/optimizer-api/pull/102 only rounds fields that were already specified as integers. Those fields are string so they are not rounded in that PR.

This PR might be merged before https://github.com/Mapotempo/optimizer-api/pull/102.
When it is done I can just round those fields and edit this documentation on https://github.com/Mapotempo/optimizer-api/pull/102 before we merge it.

Reply:
In fact I should just not change this documentation and fix it properly in PR102.

Reply:
I think it is better to give the correct type here in the first place. Also (you should test it but) i suspect grape will handle the rounding automatically, in that case we don't even need PR102. Plus since the rounding is done at the endpoint, we don't have to deal with all the issues I mentioned in PR102 (i.e., accumulating rounding errors)

Reply:
The costs are float, time and distance are integers.
With a cost per 100km of 6€ we should have a cost per meter of 0.00006€.
For a route of 56 km, the route has a cost 3.36€. Even if within the API we perform some scale change, the output should match the input expectation

Reply:
You are right I was confused the costs and stats ;) All costs effectively stay Float!